deployment/azuredeploy.json

```diff
@@ -255,6 +250,54 @@
                 "WEBSITE_RUN_FROM_PACKAGE": "1"
             }
         },
+        {

```

Comment:
Given that a lot of additional settings would now be applied to all the function apps, is it possible to move them to the common area (somewhere around line 63-67) and get rid of these blocks completely?

Reply:
It can be moved to a single block but there are a few settings which are not applicable for all function apps